15_totalIntegers/solution/totalIntegers-solution.js

```diff
@@ -0,0 +1,21 @@
+const totalIntegers = function (obj, count = 0) {
+  const isObject = (value) => typeof value === 'object' && value !== null;
```

Comment:
This helper definition should be moved outside of the recursive function's body. This is because every time this function is run, there must be a new `isObject()` function object created for the current scope. This is wasteful -- each recursive call could just be using the same `isObject()` helper function if it were defined outside this function body.

This isn't super meaningful in the context of this specific problem (ie none of the examples are going to have performance issues because of this), but it *is* good practice to avoid defining constant factors (like this helper function) inside of loops or recursive functions.